src/reversion/management/commands/deleterevisions.py

```diff
@@ -71,7 +78,14 @@ def handle(self, *app_labels, **options):
         days = options["days"]
         keep = options["keep"]
         force = options["force"]
-        confirmation = options["confirmation"]
+        interactive = options.get("interactive")
+        if not options.get("confirmation"):
+            interactive = False
+        warnings.warn(
```

Comment:
If I read this correctly, the deprecation warning will be emitted irrespective of whether the "confirmation" argument is given. I think we need something like:

``` py
if options["confirmation"] is False:
    interactive = False
    warnings.warn(  # etc...
```